app/src/marketplace/components/Modal/ChooseAccessPeriodDialog/index.jsx

```diff
@@ -25,7 +25,14 @@ export type Props = {
     dataPerUsd: ?NumberString,
     pricePerSecond: $ElementType<Product, 'pricePerSecond'>,
     priceCurrency: $ElementType<Product, 'priceCurrency'>,
-    onNext: (time: NumberString, timeUnit: TimeUnit, paymentCurrency: PaymentCurrency) => Promise<void>,
+    onNext: (
```

Comment:
Maybe consider using object parameters for readability instead of adding parameters

Reply:
It's a good idea. I tried, but had some issues with it I couldn't easily solve.